docs/guides/frontend-redux.md

```diff
@@ -0,0 +1,63 @@
+# Frontend Data Architecture
+
+This post describes the architecture used throughout [frontend-scaffold](https://github.com/Code-4-Community/frontend-scaffold), Code4Community's shared frontend template used by all client projects. The scaffold is a React application created with Create React App using the TypeScript template. It uses Redux for application state management. 
+
+## What is Redux? 
+
+Redux is a predictable state container for JavaScript apps.
+
+It helps you write applications that behave consistently, run in different environments (client, server, and native), and are easy to test. On top of that, it provides a great developer experience, such as live code editing combined with a time traveling debugger.
```

Comment:
Ooh, the time traveling debugger sounds cool

Reply:
Yeah! We really get full control over application data, which is also useful for mocking in tests - mock the redux store, and because all of your app data comes from that, you can expect things easily based on the mocked data. Without one central place, we'd never be able to mock like that, or step back and forth between states in _the entire app_.